### PR TITLE
BLD: added required "stage" field

### PR DESCRIPTION
**Pull request by lacabra**
Added new required field `stage`, which is an enum [`nominee`, `candidate`, `DPG`] to capture the stage of this product along the vetting process. The intent is to be able to differentiate and highlight those products that have been fully vetted as Digital Public Goods, see the top chart at https://digitalpublicgoods.net/platform

### CODE REVIEW

**Review by conradsp — APPROVED** [review]

**Comment by conradsp** [comment]
@lacabra - one question before I merge. As the stage is a required field, are we going to have to update all of the products and add stage? Otherwise, I think the CI will break.

**Comment by lacabra** [comment]
Yes, I'm responsible for this, so I will merge when ready.

Merging this PR here won't break because this repo does not have a CI, but propagating these changes to the `products` repo will break. I will propagate these changes at the same time I push the new field to all existing products.

**Pull request merged**
